src/index.js

```diff
@@ -553,33 +553,205 @@ export default class ToggleBlock {
   }
 
   /**
-   * Adds an event in a existent button to destroy the nested blocks
-   * when the toggle root is removed.
+   * Return the number of blocks inside the root Toggle
+   * @param {string} fk - The id of the root Toggle
+   */
+  getDecendentsNumber(fk) {
+    let counter = 0;
+    const listChildren = document.querySelectorAll(`div[foreignKey="${fk}"]`);
+    listChildren.forEach((child) => {
+      // Evaluate if the child is a toggle
+      if (child.hasAttribute('status')) {
+        const childId = child.querySelector('.toggle-block__selector').getAttribute('id');
+        counter += this.getDecendentsNumber(childId);
+      }
+      counter += 1;
+    });
+    return counter;
+  }
+
+  /**
+   * Highlight the blocks that belongs to the Toggle
+   * @param {string} fk - The id of the root Toggle
+   */
+  highlightToggleItems(fk) {
+    const listChildren = document.querySelectorAll(`div[foreignKey="${fk}"]`);
+    listChildren.forEach((child) => {
+      child.classList.add('ce-block--selected');
+      // Evaluate if the child is a toggle, then highlight also its children
+      if (child.hasAttribute('status')) {
+        const childId = child.querySelector('.toggle-block__selector').getAttribute('id');
+        this.highlightToggleItems(childId);
+      }
+    });
+  }
+
+  /**
+   * Adds events for the move up, move down and delete options in the toolbar
    */
   renderSettings() {
     const settingsBar = document.getElementsByClassName('ce-settings--opened');
     const optionsContainer = settingsBar[0];
     const options = optionsContainer.lastChild;
     const toggleIndex = this.api.blocks.getCurrentBlockIndex();
-    const listChildren = document.querySelectorAll(`div[foreignKey="${this.wrapper.id}"]`);
 
-    for (let i = 0; i < listChildren.length; i += 1) {
-      listChildren[i].classList.add('ce-block--selected');
-    }
+    this.highlightToggleItems(this.wrapper.id);
 
     setTimeout(() => {
+      const moveDownButton = options.getElementsByClassName('ce-tune-move-down')[0];
+      if (moveDownButton) {
+        moveDownButton.addEventListener('click', () => {
+          this.moveToggle(toggleIndex, 0);
+        });
+      }
+
+      const moveUpButton = options.getElementsByClassName('ce-tune-move-up')[0];
```

Comment:
I'm not sure if when you use `[0]` you have validated the list size

Reply:
I haven't validated it, but I will do it. Thanks!